ui/app/src/components/Navigation/PathNavigator/Widget.tsx

```diff
@@ -508,24 +401,6 @@ export default function (props: WidgetProps) {
     createController: path === '/scripts',
     translation: path.includes('site/website/')
   };
-
-  const exec = useCallback(
-    (action) => {
-      _dispatch(action);
-      const { type, payload } = action;
-      switch (type) {
-        case fetchPath.type:
-        case setCurrentPath.type:
-          getChildrenByPath(site, fetchPath.type === type ? state.currentPath : payload).subscribe(
-            (response) => exec(fetchPathComplete(response)),
-            (response) => dispatch(showErrorDialog({ error: response }))
-          );
-          break;
-      }
-    },
-    [dispatch, site, state]
-  );
-
   const [collapsed, setCollapsed] = useState(false);
```

Comment:
Please add expanded/collapsed as one of the redux state props so we can restore that piece of the state too.